### PR TITLE
Gender -> biologicalSex

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

Change prompted from writing up some documentation for how to format your CSV ([docs](https://docs.google.com/document/d/1iOzvBM8K-kzt5UPUcrwT1EdQZCbrJNaeCmgAypHB9XE/edit?ts=601dc348)). 

Change the CSV header from "Gender" to "biologicalSex" to match the app

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public String processPersonCSV(InputStream csvStream) throws IllegalGraphqlArgum
                     parseString(row.get("County")),
                     parseRace(row.get("Race")),
                     parseEthnicity(row.get("Ethnicity")),
-                    parseGender(row.get("Gender")),
+                    parseGender(row.get("biologicalSex")),
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java:99]: A tad ironic but not customer facing. 😺

**Review by benwarfield-usds — APPROVED** [review]
:shipit:

**Comment by jeremyzitomer-usds** [comment]
from a code perspective no concerns of course.
from a gender/sex perspective, i owe it to yall to raise the point that changing the focus from "gender" to "biological sex" is often seen as hurtful in transgender communities, without extensive justification for what about a person's "biology" is so crucial to the question at hand. it's often seen as a cop-out way to reduce something that is still non-binaristic, to a binary. i know this is outside the scope of this code review, and i don't want to block our progress on this, but i have to keep insisting that the CDC does not seem to have done their due diligence on this gender/sex question, and there are transgender/intersex people who will be harmed by that lack of proper forethought.

**Comment by jeremyzitomer-usds** [comment]
(having said that, go ahead and merge.)

**Pull request merged**